domains/ashishagarwal.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "e-coders",
+        "email": "aasheeshsaumya@gmail.com"
+    },
+    "record": {
+        "CNAME": "e-coders.github.io"
```

Comment:
site gives 404 error

Reply:
Oh...damn I forgot it I will fix it by tommorow